x/bor/keeper/veblop.go

```diff
@@ -12,14 +12,24 @@ import (
 	staketypes "github.com/0xPolygon/heimdall-v2/x/stake/types"
 )
 
-const ProducerSetLimit = uint64(3)
+const (
+	producerSetLimit    = uint64(3)
+	newProducerSetLimit = uint64(4)
+)
+
+func GetProducerSetLimit(ctx sdk.Context) uint64 {
+	if ctx.BlockHeight() >= helper.GetSetProducerDowntimeHeight() {
+		return newProducerSetLimit
+	}
+	return producerSetLimit
+}
```

Comment:
Might be a good idea to move these to https://github.com/0xPolygon/heimdall-v2/blob/develop/helper/config.go#L449-L481, where other HF params are specified.

Reply:
Done.